src/components/section/SectionHero.jsx

```diff
@@ -23,26 +23,24 @@ export function SectionHero() {
 					</Row>
 				</Container>
 			</div>
-			{connected && isGenesis !== undefined && (
```

Comment:
please keep the isGenesis condition so that the site is usable for pre-GME phase too (in case someone forks)